internal/integration/helpers.go

```diff
@@ -55,23 +55,32 @@ import (
 func NewTestServer(tb testing.TB) (*serverenv.ServerEnv, *Client) {
 	tb.Helper()
 
-	ctx := context.Background()
-
-	bs, err := storage.NewMemory(ctx)
-	if err != nil {
-		tb.Fatal(err)
+	// Do not start the server when running short tests.
+	if testing.Short() {
+		tb.SkipNow()
 	}
 
-	if v := os.Getenv("GOOGLE_CLOUD_BUCKET"); v != "" && !testing.Short() {
+	ctx := context.Background()
+
+	var bs storage.Blobstore
+	var err error
+	if v := os.Getenv("INTEGRATION_GOOGLE_CLOUD_BUCKET"); v != "" {
```

Comment:
Please also update this line: https://github.com/google/exposure-notifications-server/blob/5ddd4aa0799a029927eb9e5f74fa245d69e4edad/scripts/e2e-test.sh#L51